packages/@lwc/engine-core/src/framework/upgradable-element.ts

```diff
@@ -9,8 +9,8 @@ import { Renderer } from './renderer';
 
 type UpgradeCallback = (elm: HTMLElement) => void;
 
-export interface UpgradableCustomElementConstructor extends CustomElementConstructor {
-    upgrade: UpgradeCallback;
+interface UpgradableCustomElementConstructor extends CustomElementConstructor {
+    new (upgradeCallback?: UpgradeCallback): HTMLElement;
```

Comment:
from a static value to an optional argument

Reply:
Why is the argument optional? Shouldn't we pass a upgradeCallback to all the UpgradableElement for them to work?

Reply:
Well, `documen.createElement` will new it without arguments, that's probably a different problem that the type system has no way to understand or connect, so it will very likely be ok to making it required argument. It is just weird that something that extends HTMLElement must expect an argument, that's all.